build.cmd

```diff
@@ -1,6 +1,11 @@
 @echo off
 cls
 
+.paket\paket.bootstrapper.exe
+if errorlevel 1 (
+  exit /b %errorlevel%
+)
+
```

Comment:
There is no need to  commit `paket.bootstrapper.exe`.
Note that `paket.exe` that you removed has very similar file size
<img width="501" alt="OpenNLP_1_9_1_Upgrade_by_NightOwl888_·_Pull_Request__4_·_sergey-tihon_OpenNLP_NET" src="https://user-images.githubusercontent.com/1197905/67265903-8b040600-f4b7-11e9-844d-ad02b8bc1df2.png">

So `paket.exe` is not real Paket, it was bootstrapper used in magic mode
https://fsprojects.github.io/Paket/bootstrapper.html#Magic-mode

Reply:
No problem. The commit can easily be reverted/excluded.